Core/Model/FormaPago.php

```diff
@@ -84,6 +84,19 @@ class FormaPago extends Base\ModelClass
      */
     public $tipovencimiento;
 
+    /**
+     * Returns the bank accounts associated with the company.
+     * 
+     * @return array
+     */
+    public function getBankAccount()
+    {
+        $payAccount = new CuentaBanco();
+        if ($payAccount->loadFromCode('', [new DataBaseWhere('codcuenta', $this->codcuentabanco)])) {
```

Comment:
No hagas esto. Fíjate que solamente devuelves algo si loadfromcode devuelve true. Tienes que devolver algo siempre.

$payAccount = new CuentaBanco();
$payAccount->loadFromCode('', [new DataBaseWhere('codcuenta', $this->codcuentabanco)]);
return $payAccount;